source/creating_doc/quickstart_rst.rst

```diff
@@ -0,0 +1,139 @@
+Quick Start: Writing RST
+########################
+
+RST is very powerful, and flexible.  Below we provide a quick guide for how you
+can use it to write Open edX documentation.
+
+For more details check out the `RST Primer`_
+
+.. _RST Primer: https://docutils.sourceforge.io/docs/user/rst/quickstart.html
+
+Headings
+********
+
+.. code-block:: RST
+
+   Heading 1
+   #########
+
+   Heading 2
+   *********
+
+   Heading 3
+   =========
+
+   Heading 4
+   ---------
+
+.. note::
+   :class: dropdown
+
+   RST can be very flexible, this is the consistent way we want to do it in this
+   project.
```

Comment:
Might be worth being less vague here:
```suggestion
   RST is very flexible: it will allow you to use almost any symbol to underline headings, as long as you're consistent between heading level. However, the way shown above is how headings should be defined in all Open edX documentation. 
```